Code/Tools/TestImpactFramework/Runtime/Code/Source/TestImpactClientSequenceReportSerializer.cpp

```diff
@@ -603,4 +603,254 @@ namespace TestImpact
 
         return stringBuffer.GetString();
     }
+
+    AZStd::chrono::high_resolution_clock::time_point TimePointFromMsInt64(AZ::s64 ms)
```

Comment:
Not sure if having the type in the name of the function is necessary, that'd be implicit by the parameter. Maybe it could be `TimePointFromMs`?

Reply:
I agree that the type in the name is technically redundant but as there is a back and forth with strings and sized types for the JSON serial/deserial I think there is utility in having the size of the type signposted in the name. 

Let me have another look at the call sites with fresh eyes and get back to you on this one 👍